spec/models/food_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Food', type: :feature do
+  subject do
+    @user = User.create(name: 'Test User', role: 'admin', email: 'tests@gmail.com', password: '123456')
+    @food = Food.create(name: 'Test Food', user_id: @user.id, measurement_unit: 'kilogram', price: 100)
+  end
+
+  it 'is not valid without a name' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a measurement unit' do
+    subject.measurement_unit = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a price' do
+    subject.price = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid with a price less than 0' do
+    subject.price = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'is valid with a name, measurement unit, price, and user' do
+    subject.name = 'Test Food'
```

Comment:
- [ ] Great writing unit test but you should write integration tests using `capybara` as required in order to complete the requirements 
<img width="429" alt="Screenshot 2022-12-23 at 17 48 29" src="https://user-images.githubusercontent.com/61790587/209371947-0454ea3a-833e-4321-aa26-ffd5d8d7d30b.png">